wikimedia/wikipedia-ios
# Cleanup logging

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T260724

### Notes
* This cuts down on noise, moves everything (except testing logging) to CocoaLumberjack, and improves logging format.

### Test Steps
1. Run app. Make sure log lines look decent. Feel free to make comments if you want improvements on how logging looks.

### Screenshots
<img width="817" alt="Screenshot 4" src="https://user-images.githubusercontent.com/9295855/104080173-63dc4600-51db-11eb-9d1a-89cb809bb0f9.png">
